README.adoc

```diff
@@ -1103,8 +1103,7 @@ User.where.not('status = ? AND plan = ?', 'active', 'basic')
 
 === Redundant `all` [[redundant-all]]
 
-Using `all` as a receiver for Active Record query methods is redundant. 
-The result won't change without `all`, so it can be removed.
+When the receiver for Active Record query methods is `all`, the result won't change if `all` is omitted.
```

Comment:
Can we consider ‘delete*’ “query methods”?
The cop goes beyond just query methods.
What if we briefly mention that ‘all’ is redundant except for those not getting i to details?

Reply:
Thank you for your comment!

It's a tough call, but I've always considered `delete*` as query methods. Upon checking the `ActiveRecord::Querying::QUERYING_METHODS` constant in ActiveRecord, it includes methods like `delete*` and `update*`. Given this, in the context of ActiveRecord, it could be argued that `delete*` is also treated as a query method.

However, for those unfamiliar with these specifics, we might consider revising the phrasing.

> What if we briefly mention that ‘all’ is redundant except for those not getting into details?

What specific wording do you think would be appropriate? I'd appreciate any suggestions you might have.

Reply:
I made revisions based on the feedback. How does this look? 5efdfa894706dd826a65323decb50e1c6e6956b1